zerver/views/auth.py

```diff
@@ -34,6 +40,55 @@
 import requests
 import time
 import ujson
+import datetime
+from typing import Text
+
+def send_find_my_team_emails(user_profile):
+    # type: (UserProfile) -> None
+    path = 'zerver/emails/find_team/'
+    text_template = path + 'find_team_email.txt'
+    html_template = path + 'find_team_email_html.txt'
```

Comment:
I would probably just put the 2 paths in explicitly, it feels more greppable that way.